CONTRIBUTING.md

```diff
@@ -17,23 +17,23 @@ details how to contribute in a standardized and efficient manner.
   - Minor changes (e.g., grammatical fixes) do not require an Issue first.
 - A PR should be a minimal and accurate answer to exactly one identified and
   agreed problem. PRs that tackle multiple problems are harder to review and
-  slower to merge when the uncontroversial changes are held up by the more
+  slower to merge when uncontroversial changes are held up by the more
   discussed changes.
-- Where applicable, a PR or commit message body SHOULD reference an Issue by
-  number (e.g. Fixes #33”).
-- Use the Draft PR feature on Github or title your PR with `WIP` if your PR is
-  not ready for review immediately upon submission
+- Where applicable, a PR or commit message body should reference an Issue by
```

Comment:
This is probably a conversation the team should have. We have a bunch of keywords that are meant to mimic RFCs (see https://www.rfc-editor.org/rfc/rfc2119) and they tend to be written in all caps. This is one example, but there are more towards the end of the document. I don't think they mean anything else than what they mean in the plain English language, so, as far as I'm concerned, they can safely be demoted to normal words. However, I think it might be a good time to discuss this issue on Slack and, independently of this PR, update all occurrences to reflect what has been decided.